terminus-ui/csv-entry/src/csv-entry.component.ts

```diff
@@ -131,6 +131,11 @@ export class TsCSVEntryComponent implements OnInit, OnDestroy {
    */
   protected uid = `ts-csv-entry-${nextUniqueId++}`;
 
+  /**
+   * Define the static height needed in the DOM for the external rows
+   */
+  public fakeRowHeight = '35px';
```

Comment:
Magic number 😄

Reply:
yeah.. unfortunately, at least for now, it truly is magic 😂